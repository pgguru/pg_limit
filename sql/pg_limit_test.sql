SELECT generate_series(1,10);
SET pg_limit.max_rows = 5;
SELECT generate_series(1,10);
SET pg_limit.notify_total_count = true;
SELECT generate_series(1,10);
