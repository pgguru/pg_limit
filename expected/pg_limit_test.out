SELECT generate_series(1,10);
 generate_series 
-----------------
               1
               2
               3
               4
               5
               6
               7
               8
               9
              10
(10 rows)

SET pg_limit.max_rows = 5;
SELECT generate_series(1,10);
NOTICE:  pg_limit: result set was truncated to the first 5 rows
 generate_series 
-----------------
               1
               2
               3
               4
               5
(5 rows)

SET pg_limit.notify_total_count = true;
SELECT generate_series(1,10);
NOTICE:  pg_limit: result set was truncated to the first 5 rows (had 10 rows total)
 generate_series 
-----------------
               1
               2
               3
               4
               5
(5 rows)

